test/detect/cs/default.txt

```diff
@@ -1,11 +1,48 @@
-using System;
+using System.IO.Compression;
 
 #pragma warning disable 414, 3021
 
-/// <summary>Main task</summary>
-async Task<int, int> AccessTheWebAsync()
+namespace MyApplication
 {
-    Console.WriteLine("Hello, World!");
-    string urlContents = await getStringTask;
-    return urlContents.Length;
+    class Program : IInterface
+    {
+        /// <summary>
+        /// Main method
+        /// </summary>
+        /// <param name="args"></param>
+        static void Main(string[] args)
+        {
+            // comment
+            /* multi-lines
+             * comment
+             */
+
+            if (b == true)
+            {
+                Console.WriteLine($"Hello {Name}!");
+            }
+            else if (b == false)
+            {
+                var query = from p in entities.Persons
+                            select new { First = p.FirstName, Last = p.LastName };
+            }
+        }
+
+        private List<int> _myList;
+
+        [DataMember(Name = "mylist")]
+        public List<int> MyList
+        {
+            get
+            {
+                return _myList;
+            }
+        }
+
+        private async Task<int, int> AccessTheWebAsync()
+        {
+            string urlContents = await getStringTask;
+            return urlContents.Length;
+        }
+    }
```

Comment:
The problem with a bigger example is that it won't fit on the front page of https://highlightjs.org/, and C# is currently there. Other reasons to have shorter snippets is they make better detection tests and also that there's no point in displaying a big chunk of code to simply demonstrate highlighting. If you wanted to use it as a test for language features, they should go as separate files in test/markup/cs/ (http://highlightjs.readthedocs.io/en/latest/building-testing.html#testing-markup).